public/app/features/dashboard-scene/scene/DashboardScopesFacade.ts

```diff
@@ -0,0 +1,22 @@
+import { locationService } from '@grafana/runtime';
+import { sceneGraph } from '@grafana/scenes';
+import { ScopesFacade } from 'app/features/scopes';
+
+export interface DashboardScopesFacadeState {
+  reloadOnScopesChange?: boolean;
+  uid?: string;
+}
+
+export class DashboardScopesFacade extends ScopesFacade {
+  constructor({ reloadOnScopesChange, uid }: DashboardScopesFacadeState) {
+    super({
+      handler: (facade) => {
```

Comment:
Nit, to fix upstream in `ScopesFacade`, let's explicitly name this `onScopesChange`